### ISSUE TITLE
stdout.columns is NaN

### ISSUE THREAD

**Issue by h110m** [oven-sh/bun#1774]
### What version of Bun is running?

0.4.1

### What platform is your computer?

Microsoft Windows NT 10.0.22621.0 x64

### What steps can reproduce the bug?

Just import the readline package and try to use `readline.cursorTo(process.stdout, 0, process.stdout.collumns);`

### What is the expected behavior?

It should move the cursor to the given x and y coordinates

### What do you see instead?

```
310 |   class NodeError extends Base {
311 |     [kIsNodeError] = true;
312 |     code;
313 |     constructor(msg, opts) {
314 |
315 |       super(msg, opts);
         ^
error: The value "NaN" is invalid for argument 'y'. Reason: not specified
 code: "ERR_INVALID_ARG_VALUE"

      at new NodeError (node:readline:315:6)
      at new ERR_INVALID_ARG_VALUE (node:readline:342:4)
      at cursorTo (node:readline:1102:28)
```

**Comment by ThatOneBro** [comment]
Yeah looks like we still need to implement `process.stdout.columns` here. We should fix this soon

**Comment by h110m** [comment]
@ThatOneBro 
process.stdout.clearLine also seems to not be implemented

**Comment by Ekott2006** [comment]
Is this issue still available, I would like to work on it??

**Comment by Electroid** [comment]
Implemented in 728c8fdcdb4e9276a7bd0721bad3b5d0cbed67fa and will be available in the next release, v0.7, or now if you run: `bun upgrade --canary`
